cmd/puma-dev/main_darwin.go

```diff
@@ -169,12 +171,16 @@ func main() {
 		tlsSocketName = "SocketTLS"
 	}
 
-	fmt.Printf("! Puma dev listening on http and https\n")
+	fmt.Printf("! Puma dev running...\n")
 
-	go http.ServeTLS(tlsSocketName)
+	go func() {
+		if err := http.ServeTLS(tlsSocketName); err != nil {
+			fmt.Printf("! HTTPS Server failed: %v\n", err)
+		}
+	}()
 
 	err = http.Serve(socketName)
 	if err != nil {
-		log.Fatalf("Error listening: %s", err)
+		log.Fatalf("! HTTP Server failed: %s", err)
 	}
```

Comment:
note: we're _still_ not bailing if either https or dns fail to bind. this is the behavior we've had up until now, and it seems unnecessary to change it as part of this work.